### PR TITLE
Integrate EMI into Tablesaw

### PR DESCRIPTION
**Pull request by woodiertexas**

### CODE REVIEW

File: src/main/java/io/github/debuggyteam/tablesaw/integration/emi/TableSawEmiRecipe.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    @Override
+    public EmiRecipeCategory getCategory() {
+        return TableSawEmiIntegration.TABLESAW_CATEGORY;
+    }
+
+    @Override
+    public @Nullable Identifier getId() {
+        return null;
```
Comment (falkreon) [inline src/main/java/io/github/debuggyteam/tablesaw/integration/emi/TableSawEmiRecipe.java]: replace with

```java
return id;
```

Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/tablesaw/integration/emi/TableSawEmiRecipe.java]: done in latest push

File: src/main/java/io/github/debuggyteam/tablesaw/integration/emi/TableSawEmiRecipe.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+public class TableSawEmiRecipe implements EmiRecipe {
+    Identifier id;
+    EmiIngredient input;
+    EmiStack output;
```
Comment (Oliver-makes-code) [inline src/main/java/io/github/debuggyteam/tablesaw/integration/emi/TableSawEmiRecipe.java]: These should probably have visibility modifiers
```suggestion
    private Identifier id;
    private EmiIngredient input;
    private EmiStack output;
```

**Review by Oliver-makes-code — CHANGES_REQUESTED** [review]
Just a small thing

File: src/main/java/io/github/debuggyteam/tablesaw/integration/emi/TableSawEmiRecipe.java (hunk 2 above)
Comment (woodiertexas) [reply src/main/java/io/github/debuggyteam/tablesaw/integration/emi/TableSawEmiRecipe.java]: done in latest push

**Review by Oliver-makes-code — APPROVED** [review]
Looks good now!

**Comment by woodiertexas** [comment]
Merging as is since the EMI integration is complete. If you got any new suggestions/issues, go to the [Issues section](https://github.com/DebuggyTeam/tablesaw/issues) and make a new issue.

**Pull request merged**